x-pack/plugins/observability/public/hooks/use_values_list.ts

```diff
@@ -93,9 +95,20 @@ export const useValuesList = ({
           values: {
             terms: {
               field: sourceField,
-              size: 100,
+              size: 50,
```

Comment:
We were asked about this recently - previously we'd capped the item size to 10, now it looks like it's capped at 50. I'm unsure if there's a reason _not_ to increase this further. I didn't make this revision so I'm unsure the reasoning here, perhaps performance consideration?

Reply:
cc @andrewvc do you see any perf-related reason not to make this max higher?